src/main/java/wellnus/focus/command/CheckCommand.java

```diff
@@ -0,0 +1,98 @@
+package wellnus.focus.command;
+
+import java.util.HashMap;
+
+import wellnus.command.Command;
+import wellnus.exception.BadCommandException;
+import wellnus.focus.feature.FocusManager;
+import wellnus.focus.feature.Session;
+import wellnus.ui.TextUi;
+
+/**
+ * Represents a command to check the time left in the current session.
+ */
+public class CheckCommand extends Command {
+
+    private static final String COMMAND_KEYWORD = "check";
+    private static final int COMMAND_NUM_OF_ARGUMENTS = 1;
+    private static final String COMMAND_INVALID_ARGUMENTS_MESSAGE = "Invalid command, expected 'check'";
+    private static final String NO_ADDITIONAL_MESSAGE = "";
+    private static final String COMMAND_KEYWORD_ASSERTION = "The key should be check.";
+    private static final String CHECK_OUTPUT = "Time left: ";
+    private final Session session;
+    private final TextUi textUi;
+
+    /**
+     * Constructs a CheckCommand object.
+     * Session in FocusManager is passed into this class.
+     *
+     * @param arguments Argument-Payload Hashmap generated by CommandParser
+     * @param session   Session object which is an arraylist of Countdowns
```

Comment:
Extra whitespace before `Session`

Reply:
IDE auto formatter aligns the start of the Description, not sure if this is a concern.